learn/develop/models/index.qmd

```diff
@@ -139,7 +139,7 @@ discrim_mixture <-
   function(mode = "classification",  sub_classes = NULL) {
     # Check for correct mode
     if (mode  != "classification") {
-      rlang::abort("`mode` should be 'classification'")
+      rlang::abort("`mode` should be 'classification'.")
```

Comment:
should we encourage {cli} here?

Reply:
I didn't want to add cli styling here because I think it might not be that common yet that it's easy to read for most people - and then it just distracts from the main point. Instead, I added a reference to cli in the callout box.